CHANGELOG.md

```diff
@@ -18,6 +18,24 @@ Other improvements:
 
 Polymorphic kinds, based on the [Kind Inference for Datatypes](https://richarde.dev/papers/2020/kind-inference/kind-inference.pdf) paper (#3779, #3831, #3929, @natefaubion)
 
+Kinds are to types what types are to terms (except that kinds are actually themselves types now) but whereas we have polymorphic types, kinds were monorphic.
```

Comment:
I can't find how to insert a suggestion but I think you meant `monomorphic`. Also, I'd probably say `value` instead of `term`.

Reply:
I corrected the typo 🙇

> I'd probably say `value` instead of `term`.

I usually prefer “value” to denote normal forms, so that term/type is orthogonal to value/expression, but I’ll apply your suggestion if others find that clearer too.

Reply:
I prefer `value` as well. `term` confuses me.

Reply:
I’d agree that “term” is preferable here because it is more precise than “value”; a “term” is generally any expression, whereas a “value” generally means an expression which can’t be reduced. So `7` and `3+4` are both terms, but only the former is a value. In this case we do mean term, not value.

Reply:
Thanks, now I know the difference!